pkg/interface/src/views/apps/links/components/new.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback } from "react";
+import { RouteComponentProps } from "react-router-dom";
+import { Box, Input, Col } from "@tlon/indigo-react";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+
+import { AsyncButton } from "~/views/components/AsyncButton";
+import { FormError } from "~/views/components/FormError";
+import GroupSearch from "~/views/components/GroupSearch";
+
+import GlobalApi from "~/logic/api/global";
+import { stringToSymbol } from "~/logic/lib/util";
+import { useWaitForProps } from "~/logic/lib/useWaitForProps";
+
+import { Associations } from "~/types/metadata-update";
+import { Notebooks } from "~/types/publish-update";
+import { Groups, GroupPolicy } from "~/types/group-update";
+
+const formSchema = Yup.object({
+  name: Yup.string().required("Collection must have a name"),
+  description: Yup.string(),
+  group: Yup.string(),
+});
+
+
+export function NewScreen(props: object) {
+  const { history, api } = props;
+  const waiter = useWaitForProps(props, 5000);
+
+  const onSubmit = async (values: object, actions) => {
+    const resourceId = stringToSymbol(values.name);
+    try {
+      const { name, description, group } = values;
+      if (!!group) {
+        await props.api.graph.createManagedGraph(
+          resourceId,
+          name,
+          description,
+          group
+        );
+      } else {
+        await props.api.graph.createUnmanagedGraph(
+          resourceId,
+          name,
+          description,
+          { open: {
```

Comment:
This differs from the current behaviour of unmanaged links, which are always invite-only

Reply:
Behavior is now the same, thanks!